internal/gengapic/testdata/snippet_BidiThings.want

```diff
@@ -0,0 +1,41 @@
+func main() {
```

Comment:
Should there be an opening region tag?

Reply:
That's a good point. The existing unit tests just diff against `g.pt.String()`, so I followed that pattern, but let me see if I can compare against the entire `g.resp.File` content instead.

Reply:
I added an assertion for the header content (the start region tag). Dealing with the `g.resp.File` is messy since it isn't reset between test cases. Thanks for catching this!

Reply:
Good idea!